notebooks/2_feature_engineering.ipynb

```diff
@@ -151,23 +166,38 @@
      "name": "#%%\n"
     }
    },
+   "outputs": [],
+   "source": [
+    "# Install TextWiser\n",
+    "!pip install -q textwiser;"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
       "['Some document', 'More documents. Including multi-sentence documents.']\n",
-      "[[-0.198138    0.25826374 -0.16164406 ... -0.18344353 -0.04831419\n",
-      "   0.27247027]\n",
-      " [-0.38283753 -0.03924167 -0.10620049 ... -0.25401607  0.21510457\n",
-      "   0.4452556 ]]\n"
+      "[[-0.19813849  0.25826398 -0.16164416 ... -0.18344447 -0.04831381\n",
+      "   0.27247164]\n",
+      " [-0.38283885 -0.03924329 -0.10620081 ... -0.25401732  0.21510349\n",
+      "   0.4452555 ]]\n"
      ]
     }
    ],
    "source": [
     "# Conceptually, TextWiser is composed of an Embedding, potentially with a pretrained model,\n",
     "# that can be chained into zero or more Transformations\n",
     "from textwiser import TextWiser, Embedding, Transformation, WordOptions, PoolOptions\n",
+    "os.environ[\"TOKENIZERS_PARALLELISM\"] = \"false\"\n",
```

Comment:
I haven't had a chance to look at this yet. If we're keeping this line, can you add a comment to this saying this will be solved after a TextWiser update? Just so people aren't confused.